.gitignore

```
@@ -14,4 +87,11 @@ authawsvenv
 /mongodb
 /mongosh
 mo-expansion.yml
-tmp.json
\ No newline at end of file
+tmp.json
+haproxy.conf
+haproxy.pid
```

```suggestion
*.pid
```
To catch `.evergreen/orchestration/server.pid`